gpii/node_modules/flowManager/src/SystemUtils.js

```diff
@@ -25,3 +26,33 @@ gpii.reservedGpiiKeys = ["noUser", "restore"];
 gpii.isReservedGpiiKey = function (gpiiKey) {
     return gpii.reservedGpiiKeys.indexOf(gpiiKey) !== -1;
 };
+
+// Solve GPII-3310 by ensuring fluid.log output in the cloud lies on a single line
+gpii.renderCloudLoggingArgs = function (args) {
+    fluid.each(args, function (arg, i) {
+        var togo = fluid.isPrimitive(arg) ? arg : util.inspect(arg, {
+            breakLength: Infinity,
+            depth: 3,
```

Comment:
The depth `3` may not be deep enough to log deeply nested objects. Looking at [inspect doc](https://nodejs.org/api/util.html#util_util_inspect_object_showhidden_depth_colors), I wonder if this option value should be set to `Infinity` or `null` to log full objects.

Reply:
I'd rather wait for us to have a use case for logging an object at greater depth, rather than run the risk of blowing up some piece of logging infrastructure with a cyclic object or other overrun. Depth 3 seems like a reasonable compromise for now.